packages/replay/rollup.bundle.config.js

```diff
@@ -19,8 +18,6 @@ const baseBundleConfig = makeBaseBundleConfig({
           __SENTRY_REPLAY_VERSION__: JSON.stringify(pkg.version),
         },
       }),
-      // lodash.debounce is a CJS module, so we need to convert it to ESM first
-      commonjs(),
```

Comment:
Since build changes are always risky, I double checked NPM packages and CDN bundles and they still work with my test apps.